### PR TITLE
Prep for 2.7 beta release 🚀 

### PR DESCRIPTION
**Pull request by sydney-runkle**
Selected Reviewer: @alexmojaki

### CODE REVIEW

**Comment by sydney-runkle** [comment]
Please review

Will still add highlights from `jiter` and `pydantic-core` version bumps :).

**Review by hramezani — APPROVED** [review]
LGTM! I didn't check all the histories and I assume they are generated automatically

File: HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,133 @@
...
+* Serialize duration to hour minute second, instead of just seconds by @kakilangit in [pydantic/speedate#50](https://github.com/pydantic/speedate/pull/50)
+* Trimming str before parsing to int and float by @hungtsetse in [pydantic/pydantic-core#1203](https://github.com/pydantic/pydantic-core/pull/1203)
+
+#### Performance
+
+* `enum` validator improvements by @samuelcolvin in [#9045](https://github.com/pydantic/pydantic/pull/9045)
+* Move `enum` validation and serialization to Rust by @samuelcolvin in [#9064](https://github.com/pydantic/pydantic/pull/9064)
+* Improve schema generation for nested dataclasses by @sydney-runkle in [#9114](https://github.com/pydantic/pydantic/pull/9114)
+* Fast path for ASCII python strings by @samuelcolvin in in [pydantic/jiter#72](https://github.com/pydantic/jiter/pull/72)
+* simd int and string parsing on `aarch64` by @samuelcolvin in in [pydantic/jiter#65](https://github.com/pydantic/jiter/pull/65)
```
Comment (samuelcolvin) [inline HISTORY.md:48]:
```suggestion
* SIMD integer and string JSON parsing on `aarch64`(**Note:** SIMD on x86 will be implemented in a future release) by @samuelcolvin in in [pydantic/jiter#65](https://github.com/pydantic/jiter/pull/65)
```

File: HISTORY.md [hunk 2]
```diff
@@ -1,3 +1,133 @@
...
+* Support json cow of `jiter` by @davidhewitt in [pydantic/pydantic-core#1231](https://github.com/pydantic/pydantic-core/pull/1231)
```
Comment (samuelcolvin) [inline HISTORY.md:49]:
```suggestion
* Support JSON `Cow<str>` from `jiter` by @davidhewitt in [pydantic/pydantic-core#1231](https://github.com/pydantic/pydantic-core/pull/1231)
```

File: HISTORY.md [hunk 3]
```diff
@@ -1,3 +1,133 @@
...
+* MAJOR performance improvement: update to PyO3 0.21 final by @davidhewitt in [pydantic/pydantic-core#1248](https://github.com/pydantic/pydantic-core/pull/1248)
+* caching strings from JSON by @samuelcolvin in [pydantic/pydantic-core#1240](https://github.com/pydantic/pydantic-core/pull/1240)
```
Comment (samuelcolvin) [inline HISTORY.md:51]:
```suggestion
* cache Python strings by @samuelcolvin in [pydantic/pydantic-core#1240](https://github.com/pydantic/pydantic-core/pull/1240)
```

**Review by samuelcolvin — COMMENTED** [review]
otherwise LGTM

**Review by samuelcolvin — CHANGES_REQUESTED** [review]
It looks like https://github.com/pydantic/jiter/pull/66 isn't mentioned anywhere.

Ideally we should add some documentation for this, but that can come later.

**Pull request merged**